### ISSUE TITLE
[Question / bug] Typescript experimental decorator on class property

### ISSUE THREAD

**Issue by Tallyrald** [eslint/eslint#15182]
### Environment

Node version: 16.11.1
npm version: 8.1.0
Local ESLint version: 8.0.1
Global ESLint version: -
Operating System: Windows 10 64bit
Typescript version: 4.4.4

### What parser are you using?

@typescript-eslint/parser

### What did you do?

<details>
<summary>Configuration</summary>

```js
{
	'env': {
		'browser': true,
		'es6': true,
		'node': true,
	},
	'parser': '@typescript-eslint/parser',
	'parserOptions': {
		'project': ['./tsconfig.json'],
		'sourceType': 'module',
	},
	'plugins': [
		'@typescript-eslint',
	],
	'rules': {
		'indent': ['error', 'tab', {'SwitchCase': 1}],
	},
	'settings': {
		'import/parsers': {
		  '@typescript-eslint/parser': ['.ts', '.tsx'],
		},
		'import/resolver': {
			'typescript': {
				'directory': './tsconfig.json',
			},
		},
	},
}
```
</details>

After upgrading to eslint 8.0.X, my typescript project started throwing indent errors for the following:

```typescript
class DemonstrationClass {

	@Column({
		default: 0,
	})
	amount: number;
	/* ^^^^ Expected indentation of 2 tabs but found 1. */
}
```

On ESlint 7.32.0 the indent error does not appear.
Is this a bug or is this due to a modification/fix on the indent rule?

### What did you expect to happen?

No indent error for the property.

### What actually happened?

ESlint throws indent error for the line.

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by nzakas** [comment]
You’ll need to upgrade typescript-eslint. See https://github.com/eslint/eslint/issues/15149

If that doesn’t solve it, please follow up with the typescript-eslint project for help.
